nixos/modules/config/xdg/portal.nix

```diff
@@ -61,6 +61,17 @@ in
         Defaults to `false` to respect its opt-in nature.
       '';
     };
+
+    xdgOpenUsePortal = mkOption {
+      type = types.bool;
+      default = false;
+      description = lib.mdDoc ''
+        Sets environment variable `XDG_OPEN_USE_PORTAL` to `1`
```

Comment:
The variable XDG_OPEN_USE_PORTAL is a variable only defined in scope of nixpkgs. Should we not add some prefix to make clear is not an "official" XDG variable?

Reply:
I'll add the NIXOS prefix, when/if the upstream PR gets merged without a prefix we can update the module.

Reply:
Prefix added.